node_cli/configs/user.py

```diff
@@ -160,7 +161,8 @@ def get_validated_user_config(
 
     params = to_lower_keys(params)
     user_config = user_config_class(**params)
-    validate_user_config(user_config)
+    if not skip_usr_conf_validation:
```

Comment:
let's rename to `skip_user_conf_validation`, since we don't have `usr` anywhere in the code

Reply:
renamed